app/controllers/users_controller.rb

```diff
@@ -9,21 +9,28 @@ def create
     @user = User.new(user_params)
     token = params[:stripeToken]
 
-    if @user.save
-      handle_invitation
-      Stripe.api_key = Rails.configuration.stripe[:secret_key]
+    if @user.valid?
       begin
-        charge = Stripe::Charge.create(
-          :amount => 999, 
-          :currency => "usd",
+        charge = StripeWrapper::Charge.create({
```

Comment:
Certainly there isn't a need to use `{}` for a hash of arguments for `create` Also you are using ruby old hash syntax.